common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -158,9 +158,13 @@ class DB(contextWrapper: ContextWrapper) {
     contextWrapper.dbContext.run(quote(query[Recipe]).filter(r => (r.status == lift(RecipeStatusCurated.name) || r.status == lift(RecipeStatusVerified.name))).sortBy(r => r.publicationDate)(Ord.desc).take(1)).headOption
   }
 
-  def resetOriginalRecipesStatus(): Unit = {
-    val a = quote(query[Recipe].filter(_.status == "Pending").update(_.status -> "Ready"))
+  def resetOriginalRecipesInPendingStatuses(): Unit = {
+    val a = quote(query[Recipe].filter(_.status == lift(RecipeStatusPendingCuration.name)).update(_.status -> lift(RecipeStatusReady.name)))
```

Comment:
Ok, ok. I will correct that, in few minutes.... 
If fact the style of db calls in that file is not consistent, I will correct that as well :)